.rubocop_local.yml

```diff
@@ -0,0 +1,19 @@
+#
+# Overrides
+#
+GlobalVars:
+  AllowedVariables:
+  # Loggers
+  - $log
+  - $vim_log
+#
+# Special Exclusions
+#
+AllCops:
+  TargetRubyVersion: 2.2
```

Comment:
I removed it in https://github.com/ManageIQ/manageiq-gems-pending/pull/253 and moved it to guides in https://github.com/ManageIQ/guides/pull/238 so you no longer need it here.